rfcs/0000-slsa-attestation.md

```diff
@@ -0,0 +1,361 @@
+        }
+      },
+      {
+        "uri": "paketobuildpacks/builder-jammy-tiny",
+        "digest": {
+          "sha256": "78d74bd1c27f633341045f1c5f7f33209f6af0a5dc5700fdfd71200b5b5a0b9a"
+        },
+        "annotations": {
+          "paketo-buildpacks/go-build": "2.1.0",
+          "paketo-buildpacks/go-mod-vendor": "1.0.25",
+          "paketo-buildpacks/node-engine": "3.0.1",
+          "paketo-buildpacks/go-dist": "2.4.2",
+          "paketo-buildpacks/yarn": "1.2.0",
+          "paketo-buildpacks/liberty": "3.8.9",
+          "paketo-buildpacks/bellsoft-liberica": "10.4.2",
+          "paketo-buildpacks/spring-boot": "5.27.5",
+          "paketo-buildpacks/google-stackdriver": "8.0.3",
+          "paketo-buildpacks/ca-certificates": "3.6.6",
+          "paketo-buildpacks/ca-certificates": "3.6.5",
+          "paketo-buildpacks/apache-tomee": "1.7.7",
+          "paketo-buildpacks/apache-tomcat": "7.13.15",
+          "paketo-buildpacks/azure-application-insights": "5.17.1",
+          "paketo-buildpacks/procfile": "5.6.7",
+          "paketo-buildpacks/procfile": "5.6.6",
+          "paketo-buildpacks/java-memory-assistant": "1.4.8",
+          "paketo-buildpacks/datadog": "4.3.0",
+          "paketo-buildpacks/encrypt-at-rest": "4.5.9",
+          "paketo-buildpacks/maven": "6.15.11",
+          "paketo-buildpacks/gradle": "7.6.1",
+          "paketo-buildpacks/sbt": "6.12.9",
+          "paketo-buildpacks/clojure-tools": "2.8.12",
+          "paketo-buildpacks/leiningen": "4.6.8",
+          "paketo-buildpacks/watchexec": "2.8.6",
+          "paketo-buildpacks/watchexec": "2.8.5",
+          "paketo-buildpacks/syft": "1.39.0",
+          "paketo-buildpacks/jattach": "1.4.8",
+          "paketo-buildpacks/executable-jar": "6.8.2",
+          "paketo-buildpacks/git": "1.0.7",
+          "paketo-buildpacks/dist-zip": "5.6.7",
+          "paketo-buildpacks/environment-variables": "4.5.6",
+          "paketo-buildpacks/environment-variables": "4.5.5",
+          "paketo-buildpacks/image-labels": "4.5.5",
+          "paketo-buildpacks/image-labels": "4.5.4",
+          "paketo-buildpacks/java": "10.3.1",
+          "paketo-buildpacks/go": "4.6.0"
+        }
+      }
+    ]
+  },
+  "runDetails": {
+    "builder": {
+      "id": "https://kpack.io/signed-build",
+      "version": {
+        "kpack": "0.12.2"
+        "lifecycle": "0.17.0"
+      },
+      "builderDependencies": [
+        {
+          "name": "Namespace",
+          "content": "{\"name\":\"default\",\"resourceVersion\":\"195\"}"
+        },
+        {
+          "name": "Build",
+          "content": "{\"name\":\"test-build-1\",\"resourceVersion\":\"5\"}"
+        },
+        {
+          "name": "ServiceAccount",
+          "content": "{\"name\":\"default\",\"resourceVersion\":\"269\"}"
+        },
+        {
+          "name": "Secrets",
+          "content": "[{\"name\":\"my-registry-creds\",\"resourceVersion\":\"53\"},{\"name\":\"my-git-creds\"}]"
+        },
```

Comment:
I'm not too sold on this, I mainly included it as a way to trace an attestation back to a particular k8s object version (hence the `resourceVersion`). But I'm not too sure how practical this information will be, I guess it lets you know if the current k8s object is or is not the same as the one used during the build?

Reply:
Should we include the resource version as well?

Reply:
Good catch, don't know why I included it in the first secret but not the second